packages/react-components/react-tag-picker/library/src/components/TagPickerOption/__snapshots__/TagPickerOption.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`TagPickerOption renders a default state 1`] = `
 <div>
   <div
     aria-selected="false"
-    class="fui-TagPickerOption fui-Option"
```

Comment:
why remove `.fui-Option`? it seems like we'd be potentially breaking snapshot tests for no benefit

Reply:
Gotcha, I'll revert this and keep using `Option` styles, but I'll change the order

Reply:
oh ok I see the issue with the merging order - I believe it will still break test snapshots